asab/metrics/service.py

```diff
@@ -16,27 +16,32 @@
 
 #
 
+
 def metric_dimension(metric_name, tags):
 	dim = metric_name
 	if tags is not None:
 		for k in sorted(tags.keys()):
 			dim += ',{}={}'.format(k, tags[k])
 	return dim
 
-#
 
 class MetricsService(asab.Service):
 
 	def __init__(self, app, service_name):
 		super().__init__(app, service_name)
+		asab.Config.add_defaults({
+			"asab.metric.service": {
+				"flush_interval": 60
+			}
+		})
```

Comment:
This code has to be on the top level - otherwise it will be executed everytime I instantiate the metric service.
See ASAB docs.